web/generate-zones-config.js

```diff
@@ -14,14 +14,31 @@ const mergeZones = () => {
   const zoneFiles = fs.readdirSync(basePath);
   const filesWithDir = zoneFiles.map((file) => `${basePath}/${file}`);
 
-  const UNNECESSARY_ZONE_FIELDS = ['fallbackZoneMixes', 'isLowCarbon', 'isRenewable', 'emissionFactors'];
+  const UNNECESSARY_ZONE_FIELDS = [
+    'fallbackZoneMixes',
+    'isLowCarbon',
+    'isRenewable',
+    'emissionFactors',
+    'capacity',
+    'comment',
+    '_comment',
+  ];
   const zones = filesWithDir.reduce((zones, filepath) => {
     const zoneConfig = yaml.load(fs.readFileSync(filepath, 'utf8'));
     for (const key in zoneConfig) {
       if (UNNECESSARY_ZONE_FIELDS.includes(key)) {
         delete zoneConfig[key];
       }
     }
+    /*
+     * The parsers object is only used to check if there is a production parser in the frontend.
+     * This moves this check to the build step, so we can minimize the size of the frontend bundle.
+     */
+    if (zoneConfig?.parsers?.production?.length > 0) {
+      zoneConfig.parsers = true;
+    } else {
+      zoneConfig.parsers = false;
```

Comment:
We could remove the parser object entirely for some additional size savings if we want.